r2dbc/core/src/main/java/com/google/cloud/sql/core/GcpConnectionFactoryProvider.java

```diff
@@ -146,7 +146,7 @@ public ConnectionFactory create(ConnectionFactoryOptions connectionFactoryOption
             .build();
     // Precompute SSL Data to trigger the initial refresh to happen immediately,
     // and ensure enableIAMAuth is set correctly.
-    InternalConnectorRegistry.getInstance().getConnectionMetadata(config);
+    // InternalConnectorRegistry.getInstance().getConnectionMetadata(config);
```

Comment:
can we just remove these three lines instead of commenting it out?

Reply:
Yes. Sorry, that fix went into the wrong PR.